data/resume/resume.json

```diff
@@ -5,48 +5,15 @@
     "label": "Frontend Software Engineer",
     "email": "jerry.batorski@gmail.com",
     "url": "https://jarekb84.github.io/me/",
-    "summaryOfQualifications": [
-      "11+ YOE as a software engineer, with a focus on the frontend",
-      "9+ YOE breaking down projects, communicating with stakeholders, delivering business value, and adapting to change",
-      "3+ YOE leading a frontend focused squad of software engineers"
-    ]
+    "summary": "Frontend focused software engineer, with over 11 years of experience, covering everything from implementing ideas to leading projects."
   },
   "work": [
     {
       "name": "Grubhub",
       "location": "Chicago, IL",
       "url": "https://www.grubhub.com/",
-      "startDate": "2016-11-01",
-      "skills": [
```

Comment:
Going with the idea that these are the only skills I want to emphasize to companies, elevated this list to the normal resume json schema structure.

The other skills for past jobs are all removed, not real incentive for me to land Angualr or C# jobs.